resources/js/components/SignIn.js

```diff
@@ -1,292 +1,250 @@
+    <Grid container component="main" sx={{ height: '100vh' }}>
+      {props.requestError === false ? (
+        <Dialog
+          fullWidth
+          open={true}
+          maxWidth="xs"
+          sx={{ marginTop: '-5%', backdropFilter: 'blur(5px)' }}
+          align="center"
+        >
+          <DialogTitle>
+            Signing In...
+            <CircularProgress
+              color="primary"
+              sx={{
+                float: 'right',
+                marginRight: '90px',
+                marginTop: '-5px'
+              }}
+            ></CircularProgress>
+          </DialogTitle>
+        </Dialog>
+      ) : null}
+      <Grid
+        item
+        xs={false}
+        sm={4}
+        md={7}
+        sx={{
+          backgroundImage: 'url(images/bg-6.jpeg)',
+          backgroundRepeat: 'no-repeat',
+          backgroundColor: (t) =>
+            t.palette.mode === 'light'
+              ? t.palette.grey[50]
+              : t.palette.grey[900],
+          backgroundSize: 'cover',
+          backgroundPosition: 'center'
+        }}
+      ></Grid>
+      <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+        <Box
+          sx={{
+            my: 8,
+            mx: 4,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center'
+          }}
+        >
+          <Avatar
+            sx={{
+              width: '200px',
+              height: '200px',
+              marginBottom: '-30px',
+              marginTop: '-50px'
+            }}
+            src="images/e-learning.png"
+          ></Avatar>
+          <Typography component="h1" variant="h5" sx={{ marginTop: '-40px' }}>
+            Sign in
+          </Typography>
+          <Typography component="h6" variant="h6" style={{ color: 'red' }}>
+            {props.requestErrorMessage}
+            &nbsp;
+          </Typography>
+          <Box
+            component="form"
+            noValidate
+            onSubmit={handleSubmit}
+            sx={{ marginTop: '10px' }}
+          >
+            <TextField
+              onKeyUp={(e) => {
+                props.validateEmail(e.target.value);
+              }}
+              id="email"
+              type="email"
+              helperText={props.emailError === '' ? ' ' : props.emailError}
+              error={props.emailError === '' ? false : true}
+              margin="dense"
+              required
+              fullWidth
+              label="Email Address"
+              name="email"
+              autoComplete="email"
+              InputLabelProps={{ required: false }}
+              autoComplete="off"
+            />
+            <TextField
+              onKeyUp={(e) => {
+                props.validatePassword('signIn', e.target.value);
+              }}
+              helperText={
+                props.passwordError === '' ? ' ' : props.passwordError
+              }
+              error={props.passwordError === '' ? false : true}
```

Comment:
```suggestion
              error={props.passwordError === ''}
```

Reply:
Removing the `=== '' ? false : true` the error will not highlight the text field as error. `error={}` should be a true or false.

Reply:
@michaelsilverio oki u can disregard this as well